main/inc/lib/document.lib.php

```diff
@@ -5361,7 +5361,7 @@ public static function create_document_link(
 
         // Get the title or the basename depending on what we're using
         if ($document_data['title'] != '') {
-            $title = $document_data['title'];
+            $title = htmlspecialchars($document_data['title']);
```

Comment:
We will have to watch this one and make sure it does not have unintended consequences... (that's changing the normal Chamilo behaviour). @jmontoyaa wasn't there something about this and accentuated characters with @baelmyhu or someone like that? That rings a bell to me...

Reply:
This may not be necessary, can't remember why I added it. This feature was developed a year and a half ago for 1.9.6 version, may not be necessary for 1.11, can't find any ussage for it as of now.

Reply:
ok, so let's remove it and add it back later on if necessary